tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -96,9 +94,9 @@ public function testGetGroupsExpanded()
     {
         $token = $this->getAccessToken('admin', 'admin');
         $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
-        $ocis->createGroup("violin-haters", "sss");
+        $ocis->createGroup("physics-lovers", "sss");
```

Comment:
write some meaning ful description
```suggestion
        $ocis->createGroup("physics-lovers", "physics lover group");
```